.github/workflows/patch-code-of-conduct.yml

```
@@ -0,0 +1,31 @@
+name: Patch CoC
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  patch:
+    runs-on: ubuntu-latest
+    name: Patch
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Patch CoC
+        uses: pkgjs/patch-my-code-of-conduct@v1.2.0
+        with:
+          base_url: '../../conduct/artifacts/CODE_OF_CONDUCT_PATCH.patch'
+          template_url: 'https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md'
```

```suggestion
          template_url: 'https://www.contributor-covenant.org/version/2/1/code_of_conduct/code_of_conduct.md'
```

Please revert this change. PR #1007 was carefully crafted to minimise our diff compared to the current upstream 2.1, where a bunch of whitespace changes have been made compared to the latest 2.0. All of that noise is now included in the patch file (which also needs to be reverted).

It's honestly fine for the script to take the latest 2.1, and turn it into our 2.0 so that all that's left in the patch is actual relevant content. This allows us to clearly see the exact points of divergence, and to have conversations focused on just those aspects.